src/main/java/uk/yermak/audiobookconverter/fx/FilesController.java

```diff
@@ -150,6 +150,17 @@ public void initialize() {
         filesChapters.getTabs().remove(chaptersTab);
 
         bookStructure.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
+        bookStructure.getSelectionModel().getSelectedItems().addListener((ListChangeListener<TreeItem<Organisable>>) c -> {
+            List<MediaInfo> list = ConverterApplication.getContext().getSelectedMedia();
```

Comment:
Question more to myself... If selected media used only for playback is there any better place to keep them or should this var have better name?
I personally though that this feature will require media player modification, but that's interesting if it works this way...

Reply:
Name seems to reflect the purpose - list of selected media items.